### ISSUE TITLE
[Quest][Northrend] The Last Line Of Defense - Spawns + Reactbehaviour

### ISSUE THREAD

**Issue by depressedme** [dalaranwow/dalaran-wow#4262]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Hello, tried to do the quest: "The Last Line Of Defense", but the mobs kept evading, and phased out. Then I removed the quest from my log, and now every NPC at "The Argent Vanguard" is gone.
I then tried to fly away but got stuck.
**Expected behaviour**: Tell us what should happen instead

The mobs should be able to be killed, with the cannon, and The Argent Vanguard should have its NPC

**Steps to reproduce the problem**:

1. Take the quest
2. 
3. 

**Include proofs for this behaviour**
My quest log:
https://gyazo.com/e809884e37c86afa3e49ee355b9b6d3c
Mob phasing out:
https://gyazo.com/9eab069ea93d936ad45080cc0b903353
Getting stuck in the air, needed to relog and die for it to stop:
https://gyazo.com/9afc7d1925ce8b9ff071509cbabcf3e9

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

Couldn't find the ID on the mob: "Frostbrood Destroyer", "Forgotten Depths Slayer" or the quest object "Argent Cannon" (I most likely have no idea where to find the ID, didn't seem to be there when I searched)

**Include Screenshots from the issue if necessary**

**Comment by depressedme** [comment]
I found a way around this, since apparently the dragon spawning in the northerne part, and one in the southwestern part, was hitable. I had to Alt + F4 3-4 times, and if I didn't I would just be stuck. Mobs are still running around randomly, and are either evading or phasing out. So yeah...

**Comment by Rushor** [comment]
Part of this is a dulicate of https://github.com/dalaranwow/dalaran-wow/issues/2878

I will make some rework for the creaturespawn.

**Comment by Rushor** [comment]
DB/Quest: The Last Line of Defence - Add spawnscript for questcreatures (they should no longer evade)

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4262
* commit: 82a399e207b5a747dd09eb35a7e894b1a2d65e92

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
